interface.yaml

```diff
@@ -0,0 +1,12 @@
+name: bind-client
+summary: Interface for intergrating bind
+maintainer: DevOps Centres
```

Comment:
This should have an email address. You may want to put your name and canonical's email address since the DevOps Centre does not have one.